lib/moonshot/build_mechanism/script.rb

```diff
@@ -41,7 +41,7 @@ def build_hook(version)
 
   def post_build_hook(_version)
     unless File.exist?(@output_file) # rubocop:disable GuardClause
-      raise Thor::Error, 'Build command did not produce output file!'
+      step.failure 'Build command did not produce output file!'
```

Comment:
Oops, you're right. I'm fixing the CF resources I'm using, then I'll test it out again.